File: VB/Task.vb
Imports DevExpress.Mvvm

Namespace E2042

    Public Class Task
        Inherits BindableBase

        Public Property Num As Integer
            Get
                Return GetValue(Of Integer)()
            End Get

            Set(ByVal value As Integer)
                SetValue(value)
            End Set
        End Property

        Public Property Name As String
            Get
                Return GetValue(Of String)()
            End Get

            Set(ByVal value As String)
                SetValue(value)
            End Set
        End Property

        Public Property IsCompleted As Boolean
            Get
                Return GetValue(Of Boolean)()
            End Get

            Set(ByVal value As Boolean)
                SetValue(value)
            End Set
        End Property
    End Class
End Namespace

File: CS/Task.cs
using DevExpress.Mvvm;

namespace E2042 {
    public class Task : BindableBase {
        public int Num { get { return GetValue<int>(); } set { SetValue(value); } }
        public string Name { get { return GetValue<string>(); } set { SetValue(value); } }
        public bool IsCompleted { get { return GetValue<bool>(); } set { SetValue(value); } }
    }
}
